src/Http/Http.Features/src/IHttpBufferingFeature.cs

```diff
@@ -1,8 +1,11 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.AspNetCore.Http.Features
 {
+    [Obsolete("See IHttpRequestBodyFeature or IHttpResponseBodyFeature DisableBuffering", error: true)]
     public interface IHttpBufferingFeature
```

Comment:
Shouldn't this show up in the ref file as obsolete too?

Reply:
It does. No?